ci-operator/step-registry/ipi/conf/aws/oidc-creds/ipi-conf-aws-oidc-creds-commands.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+PREFIX="${SHARED_DIR}/manifest"
+cat >> "${PREFIX}_cluster-authentication-02-config.yaml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: https://<something>-oidc-discovery.s3.amazonaws.com
```

Comment:
will the URL here be replaced entirely? i'm asking because of the subtle differences with us-east-1 and other regions for S3 buckets URLs.

Reply:
Yeah, what ever the url is, just here for the placeholder for now.